symplyphysics/laws/optics/spherical_lens_focus_distance.py

```diff
@@ -0,0 +1,48 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression,
+    validate_input, validate_output)
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.laws.optics import lens_focus_from_object_and_image as focus_law
+
+# Description
+## According to the definition of the focus of an ellipse,
+## the center of which is at the origin, the focus has coordinates F(c, 0),
+## where c^2 = a^2 + b^2 ("a" and "b" are the main semi-axes of the ellipse).
+## For the special case of an ellipse (circle): a = b = R, respectively,
+## the focal length in projection on any of the axes of the coordinate system:
+
+# Law: F = R/2
+# Where:
+## F - focus distance for spherical lens
+## R - curvature radius of lens
+
+focus_distance = Symbol("focus_distance", units.length)
+radius_lens = Symbol("radius_lens", units.length)
+
+law = Eq(focus_distance, radius_lens / 2)
+
+# The focus of a spherical mirror is at a point from which the distance to the object
```

Comment:
I do not see, why distance to the object equals to curvature radius. Maybe you can extend proof to explain it?